web/tests/repositories/transactionsRepository.test.ts

```
@@ -0,0 +1,101 @@
+      },
+    ]
+
+    const chain: any = {
+      select: vi.fn().mockReturnThis(),
+      eq: vi.fn().mockReturnThis(),
+      gte: vi.fn().mockReturnThis(),
+      lt: vi.fn().mockReturnThis(),
+    }
+    chain.order = vi.fn().mockReturnValue(chain)
+    // Make the builder thenable so `await query` resolves to `{ data, error }`
+    chain.then = (resolve: any) => resolve(ok(records))
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)